keps/sig-storage/20190122-csi-inline-volumes.md

```diff
@@ -312,11 +320,6 @@ type InlineVolumeSource struct {
 	// Only CSIVolumeSource can be set.
 	// Required.
 	CSIVolumeSource v1.VolumeSource
-
-	// Namespace of the pod with in-line volume. It is used to resolve
-	// references to Secrets in VolumeSource.
-	// Required.
-	Namespace string
 }
 ```
 The following steps outline how the API is used to track attachment and detachment of volumes:
```

Comment:
Can you remove the line below that says external attacher may need secret permissions

Reply:
Updated.